### PR TITLE
Implement The Game : Guess The Number.

### PR DESCRIPTION
**Pull request by bhavya-sarraf-26**
If merged this pull request will add a .cpp file that contains the implementation of The Game : Guess The Number in C++.

### CODE REVIEW

File: C++ Programs/GuessTheNumber.cpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+// To return the difference between guessing number and the guessed number.
+int checkNumDifference(int guessingNumber, int guessedNumber) {
+	return (guessingNumber-guessedNumber);
+}
+
+// To return the number to be guessed.
+int generateGuessingNumber() {
+	srand((unsigned)time(0));
```
Comment (PrajwalCH) [inline C++ Programs/GuessTheNumber.cpp:23]: Place srand(1) function ton main() just before calling importantInstructions()

**Comment by thenishantsapkota** [comment]
Kindly fetch upstream and add the file into `Hacktoberfest-2021/C++-Programs`. 
`C++ Programs` was renamed to `C++-Programs`.

File: C++ Programs/GuessTheNumber.cpp (hunk 1 above)
Comment (bhavya-sarraf-26) [reply C++ Programs/GuessTheNumber.cpp:23]: Created a new pull request (making a new branch : NewGuessTheNumber) and made the changes as told.

**Comment by bhavya-sarraf-26** [comment]
Created a new pull request (making a new branch : NewGuessTheNumber) and made the changes as told.

**Pull request merged**